2023-06-19 01:52:46,764 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-06-20 00:13:37,271 - ERROR - MainThread -fsk_hive_connect.py - execute_sql - 76 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while processing statement: FAILED: Execution Error, return code 30041 from org.apache.hadoop.hive.ql.exec.spark.SparkTask. Failed to create Spark client for Spark session e2d768df-178e-49cb-9868-26dca3a7c337: java.util.concurrent.TimeoutException: Client 'e2d768df-178e-49cb-9868-26dca3a7c337' timed out waiting for connection from the Remote Spark Driver:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:runQuery:SQLOperation.java:258', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:293', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.metadata.HiveException:Failed to create Spark client for Spark session e2d768df-178e-49cb-9868-26dca3a7c337: java.util.concurrent.TimeoutException: Client 'e2d768df-178e-49cb-9868-26dca3a7c337' timed out waiting for connection from the Remote Spark Driver:28:12", 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:getHiveException:SparkSessionImpl.java:296', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:open:SparkSessionImpl.java:140', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionManagerImpl:getSession:SparkSessionManagerImpl.java:131', 'org.apache.hadoop.hive.ql.exec.spark.SparkUtilities:getSparkSession:SparkUtilities.java:128', 'org.apache.hadoop.hive.ql.exec.spark.SparkTask:execute:SparkTask.java:122', 'org.apache.hadoop.hive.ql.exec.Task:executeTask:Task.java:199', 'org.apache.hadoop.hive.ql.exec.TaskRunner:runSequential:TaskRunner.java:97', 'org.apache.hadoop.hive.ql.Driver:launchTask:Driver.java:2250', 'org.apache.hadoop.hive.ql.Driver:execute:Driver.java:1893', 'org.apache.hadoop.hive.ql.Driver:runInternal:Driver.java:1613', 'org.apache.hadoop.hive.ql.Driver:run:Driver.java:1332', 'org.apache.hadoop.hive.ql.Driver:run:Driver.java:1327', 'org.apache.hive.service.cli.operation.SQLOperation:runQuery:SQLOperation.java:256', "*java.lang.RuntimeException:Timed out waiting for Remote Spark Driver to connect to HiveServer2.\nPossible reasons include network issues, errors in remote driver, cluster has no available resources, etc.\nPlease check YARN or Spark driver's logs for further information.:32:5", 'org.apache.hive.spark.client.SparkClientImpl:<init>:SparkClientImpl.java:124', 'org.apache.hive.spark.client.SparkClientFactory:createClient:SparkClientFactory.java:90', 'org.apache.hadoop.hive.ql.exec.spark.RemoteHiveSparkClient:createRemoteClient:RemoteHiveSparkClient.java:104', 'org.apache.hadoop.hive.ql.exec.spark.RemoteHiveSparkClient:<init>:RemoteHiveSparkClient.java:100', 'org.apache.hadoop.hive.ql.exec.spark.HiveSparkClientFactory:createHiveSparkClient:HiveSparkClientFactory.java:76', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:open:SparkSessionImpl.java:135', "*java.util.concurrent.ExecutionException:java.util.concurrent.TimeoutException: Client 'e2d768df-178e-49cb-9868-26dca3a7c337' timed out waiting for connection from the Remote Spark Driver:33:1", 'io.netty.util.concurrent.AbstractFuture:get:AbstractFuture.java:41', 'org.apache.hive.spark.client.SparkClientImpl:<init>:SparkClientImpl.java:103', "*java.util.concurrent.TimeoutException:Client 'e2d768df-178e-49cb-9868-26dca3a7c337' timed out waiting for connection from the Remote Spark Driver:8:6", 'org.apache.hive.spark.client.rpc.RpcServer$2:run:RpcServer.java:178', 'io.netty.util.concurrent.PromiseTask$RunnableAdapter:call:PromiseTask.java:38', 'io.netty.util.concurrent.ScheduledFutureTask:run:ScheduledFutureTask.java:120', 'io.netty.util.concurrent.AbstractEventExecutor:safeExecute:AbstractEventExecutor.java:163', 'io.netty.util.concurrent.SingleThreadEventExecutor:runAllTasks:SingleThreadEventExecutor.java:403', 'io.netty.channel.nio.NioEventLoop:run:NioEventLoop.java:463', 'io.netty.util.concurrent.SingleThreadEventExecutor$5:run:SingleThreadEventExecutor.java:858'], sqlState='42000', errorCode=30041, errorMessage="Error while processing statement: FAILED: Execution Error, return code 30041 from org.apache.hadoop.hive.ql.exec.spark.SparkTask. Failed to create Spark client for Spark session e2d768df-178e-49cb-9868-26dca3a7c337: java.util.concurrent.TimeoutException: Client 'e2d768df-178e-49cb-9868-26dca3a7c337' timed out waiting for connection from the Remote Spark Driver"), operationHandle=None)
2023-06-20 00:18:39,102 - ERROR - MainThread -fsk_hive_connect.py - execute_sql - 76 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while processing statement: FAILED: Execution Error, return code 30041 from org.apache.hadoop.hive.ql.exec.spark.SparkTask. Failed to create Spark client for Spark session 8f67eecd-8220-4344-9694-a8ac6b311f63: java.util.concurrent.TimeoutException: Client '8f67eecd-8220-4344-9694-a8ac6b311f63' timed out waiting for connection from the Remote Spark Driver:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:runQuery:SQLOperation.java:258', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:293', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.metadata.HiveException:Failed to create Spark client for Spark session 8f67eecd-8220-4344-9694-a8ac6b311f63: java.util.concurrent.TimeoutException: Client '8f67eecd-8220-4344-9694-a8ac6b311f63' timed out waiting for connection from the Remote Spark Driver:28:12", 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:getHiveException:SparkSessionImpl.java:296', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:open:SparkSessionImpl.java:140', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionManagerImpl:getSession:SparkSessionManagerImpl.java:131', 'org.apache.hadoop.hive.ql.exec.spark.SparkUtilities:getSparkSession:SparkUtilities.java:128', 'org.apache.hadoop.hive.ql.exec.spark.SparkTask:execute:SparkTask.java:122', 'org.apache.hadoop.hive.ql.exec.Task:executeTask:Task.java:199', 'org.apache.hadoop.hive.ql.exec.TaskRunner:runSequential:TaskRunner.java:97', 'org.apache.hadoop.hive.ql.Driver:launchTask:Driver.java:2250', 'org.apache.hadoop.hive.ql.Driver:execute:Driver.java:1893', 'org.apache.hadoop.hive.ql.Driver:runInternal:Driver.java:1613', 'org.apache.hadoop.hive.ql.Driver:run:Driver.java:1332', 'org.apache.hadoop.hive.ql.Driver:run:Driver.java:1327', 'org.apache.hive.service.cli.operation.SQLOperation:runQuery:SQLOperation.java:256', "*java.lang.RuntimeException:Timed out waiting for Remote Spark Driver to connect to HiveServer2.\nPossible reasons include network issues, errors in remote driver, cluster has no available resources, etc.\nPlease check YARN or Spark driver's logs for further information.:32:5", 'org.apache.hive.spark.client.SparkClientImpl:<init>:SparkClientImpl.java:124', 'org.apache.hive.spark.client.SparkClientFactory:createClient:SparkClientFactory.java:90', 'org.apache.hadoop.hive.ql.exec.spark.RemoteHiveSparkClient:createRemoteClient:RemoteHiveSparkClient.java:104', 'org.apache.hadoop.hive.ql.exec.spark.RemoteHiveSparkClient:<init>:RemoteHiveSparkClient.java:100', 'org.apache.hadoop.hive.ql.exec.spark.HiveSparkClientFactory:createHiveSparkClient:HiveSparkClientFactory.java:76', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:open:SparkSessionImpl.java:135', "*java.util.concurrent.ExecutionException:java.util.concurrent.TimeoutException: Client '8f67eecd-8220-4344-9694-a8ac6b311f63' timed out waiting for connection from the Remote Spark Driver:33:1", 'io.netty.util.concurrent.AbstractFuture:get:AbstractFuture.java:41', 'org.apache.hive.spark.client.SparkClientImpl:<init>:SparkClientImpl.java:103', "*java.util.concurrent.TimeoutException:Client '8f67eecd-8220-4344-9694-a8ac6b311f63' timed out waiting for connection from the Remote Spark Driver:8:6", 'org.apache.hive.spark.client.rpc.RpcServer$2:run:RpcServer.java:178', 'io.netty.util.concurrent.PromiseTask$RunnableAdapter:call:PromiseTask.java:38', 'io.netty.util.concurrent.ScheduledFutureTask:run:ScheduledFutureTask.java:120', 'io.netty.util.concurrent.AbstractEventExecutor:safeExecute:AbstractEventExecutor.java:163', 'io.netty.util.concurrent.SingleThreadEventExecutor:runAllTasks:SingleThreadEventExecutor.java:403', 'io.netty.channel.nio.NioEventLoop:run:NioEventLoop.java:463', 'io.netty.util.concurrent.SingleThreadEventExecutor$5:run:SingleThreadEventExecutor.java:858'], sqlState='42000', errorCode=30041, errorMessage="Error while processing statement: FAILED: Execution Error, return code 30041 from org.apache.hadoop.hive.ql.exec.spark.SparkTask. Failed to create Spark client for Spark session 8f67eecd-8220-4344-9694-a8ac6b311f63: java.util.concurrent.TimeoutException: Client '8f67eecd-8220-4344-9694-a8ac6b311f63' timed out waiting for connection from the Remote Spark Driver"), operationHandle=None)
2023-06-21 01:30:54,179 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-06-22 01:25:04,114 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-06-23 01:15:51,244 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-06-24 01:14:37,985 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-06-25 01:18:03,121 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-06-26 01:15:58,332 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-06-27 01:13:22,336 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-06-28 01:07:38,639 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-06-29 00:52:58,806 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-06-30 01:29:08,837 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-01 01:10:15,186 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-02 01:27:54,039 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-03 01:23:16,204 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-04 00:21:08,252 - ERROR - MainThread -fsk_hive_connect.py - execute_sql - 76 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while processing statement: FAILED: Execution Error, return code 30041 from org.apache.hadoop.hive.ql.exec.spark.SparkTask. Failed to create Spark client for Spark session e4b89e6c-bc99-41ab-9751-b99fb7d54320: java.util.concurrent.TimeoutException: Client 'e4b89e6c-bc99-41ab-9751-b99fb7d54320' timed out waiting for connection from the Remote Spark Driver:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:runQuery:SQLOperation.java:258', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:293', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.metadata.HiveException:Failed to create Spark client for Spark session e4b89e6c-bc99-41ab-9751-b99fb7d54320: java.util.concurrent.TimeoutException: Client 'e4b89e6c-bc99-41ab-9751-b99fb7d54320' timed out waiting for connection from the Remote Spark Driver:28:12", 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:getHiveException:SparkSessionImpl.java:296', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:open:SparkSessionImpl.java:140', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionManagerImpl:getSession:SparkSessionManagerImpl.java:131', 'org.apache.hadoop.hive.ql.exec.spark.SparkUtilities:getSparkSession:SparkUtilities.java:128', 'org.apache.hadoop.hive.ql.exec.spark.SparkTask:execute:SparkTask.java:122', 'org.apache.hadoop.hive.ql.exec.Task:executeTask:Task.java:199', 'org.apache.hadoop.hive.ql.exec.TaskRunner:runSequential:TaskRunner.java:97', 'org.apache.hadoop.hive.ql.Driver:launchTask:Driver.java:2250', 'org.apache.hadoop.hive.ql.Driver:execute:Driver.java:1893', 'org.apache.hadoop.hive.ql.Driver:runInternal:Driver.java:1613', 'org.apache.hadoop.hive.ql.Driver:run:Driver.java:1332', 'org.apache.hadoop.hive.ql.Driver:run:Driver.java:1327', 'org.apache.hive.service.cli.operation.SQLOperation:runQuery:SQLOperation.java:256', "*java.lang.RuntimeException:Timed out waiting for Remote Spark Driver to connect to HiveServer2.\nPossible reasons include network issues, errors in remote driver, cluster has no available resources, etc.\nPlease check YARN or Spark driver's logs for further information.:32:5", 'org.apache.hive.spark.client.SparkClientImpl:<init>:SparkClientImpl.java:124', 'org.apache.hive.spark.client.SparkClientFactory:createClient:SparkClientFactory.java:90', 'org.apache.hadoop.hive.ql.exec.spark.RemoteHiveSparkClient:createRemoteClient:RemoteHiveSparkClient.java:104', 'org.apache.hadoop.hive.ql.exec.spark.RemoteHiveSparkClient:<init>:RemoteHiveSparkClient.java:100', 'org.apache.hadoop.hive.ql.exec.spark.HiveSparkClientFactory:createHiveSparkClient:HiveSparkClientFactory.java:76', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:open:SparkSessionImpl.java:135', "*java.util.concurrent.ExecutionException:java.util.concurrent.TimeoutException: Client 'e4b89e6c-bc99-41ab-9751-b99fb7d54320' timed out waiting for connection from the Remote Spark Driver:33:1", 'io.netty.util.concurrent.AbstractFuture:get:AbstractFuture.java:41', 'org.apache.hive.spark.client.SparkClientImpl:<init>:SparkClientImpl.java:103', "*java.util.concurrent.TimeoutException:Client 'e4b89e6c-bc99-41ab-9751-b99fb7d54320' timed out waiting for connection from the Remote Spark Driver:8:6", 'org.apache.hive.spark.client.rpc.RpcServer$2:run:RpcServer.java:178', 'io.netty.util.concurrent.PromiseTask$RunnableAdapter:call:PromiseTask.java:38', 'io.netty.util.concurrent.ScheduledFutureTask:run:ScheduledFutureTask.java:120', 'io.netty.util.concurrent.AbstractEventExecutor:safeExecute:AbstractEventExecutor.java:163', 'io.netty.util.concurrent.SingleThreadEventExecutor:runAllTasks:SingleThreadEventExecutor.java:403', 'io.netty.channel.nio.NioEventLoop:run:NioEventLoop.java:463', 'io.netty.util.concurrent.SingleThreadEventExecutor$5:run:SingleThreadEventExecutor.java:858'], sqlState='42000', errorCode=30041, errorMessage="Error while processing statement: FAILED: Execution Error, return code 30041 from org.apache.hadoop.hive.ql.exec.spark.SparkTask. Failed to create Spark client for Spark session e4b89e6c-bc99-41ab-9751-b99fb7d54320: java.util.concurrent.TimeoutException: Client 'e4b89e6c-bc99-41ab-9751-b99fb7d54320' timed out waiting for connection from the Remote Spark Driver"), operationHandle=None)
2023-07-04 00:26:09,711 - ERROR - MainThread -fsk_hive_connect.py - execute_sql - 76 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while processing statement: FAILED: Execution Error, return code 30041 from org.apache.hadoop.hive.ql.exec.spark.SparkTask. Failed to create Spark client for Spark session 0db2de9d-69b6-449b-bb3b-b64635956e7b: java.util.concurrent.TimeoutException: Client '0db2de9d-69b6-449b-bb3b-b64635956e7b' timed out waiting for connection from the Remote Spark Driver:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:runQuery:SQLOperation.java:258', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:293', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.metadata.HiveException:Failed to create Spark client for Spark session 0db2de9d-69b6-449b-bb3b-b64635956e7b: java.util.concurrent.TimeoutException: Client '0db2de9d-69b6-449b-bb3b-b64635956e7b' timed out waiting for connection from the Remote Spark Driver:28:12", 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:getHiveException:SparkSessionImpl.java:296', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:open:SparkSessionImpl.java:140', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionManagerImpl:getSession:SparkSessionManagerImpl.java:131', 'org.apache.hadoop.hive.ql.exec.spark.SparkUtilities:getSparkSession:SparkUtilities.java:128', 'org.apache.hadoop.hive.ql.exec.spark.SparkTask:execute:SparkTask.java:122', 'org.apache.hadoop.hive.ql.exec.Task:executeTask:Task.java:199', 'org.apache.hadoop.hive.ql.exec.TaskRunner:runSequential:TaskRunner.java:97', 'org.apache.hadoop.hive.ql.Driver:launchTask:Driver.java:2250', 'org.apache.hadoop.hive.ql.Driver:execute:Driver.java:1893', 'org.apache.hadoop.hive.ql.Driver:runInternal:Driver.java:1613', 'org.apache.hadoop.hive.ql.Driver:run:Driver.java:1332', 'org.apache.hadoop.hive.ql.Driver:run:Driver.java:1327', 'org.apache.hive.service.cli.operation.SQLOperation:runQuery:SQLOperation.java:256', "*java.lang.RuntimeException:Timed out waiting for Remote Spark Driver to connect to HiveServer2.\nPossible reasons include network issues, errors in remote driver, cluster has no available resources, etc.\nPlease check YARN or Spark driver's logs for further information.:32:5", 'org.apache.hive.spark.client.SparkClientImpl:<init>:SparkClientImpl.java:124', 'org.apache.hive.spark.client.SparkClientFactory:createClient:SparkClientFactory.java:90', 'org.apache.hadoop.hive.ql.exec.spark.RemoteHiveSparkClient:createRemoteClient:RemoteHiveSparkClient.java:104', 'org.apache.hadoop.hive.ql.exec.spark.RemoteHiveSparkClient:<init>:RemoteHiveSparkClient.java:100', 'org.apache.hadoop.hive.ql.exec.spark.HiveSparkClientFactory:createHiveSparkClient:HiveSparkClientFactory.java:76', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:open:SparkSessionImpl.java:135', "*java.util.concurrent.ExecutionException:java.util.concurrent.TimeoutException: Client '0db2de9d-69b6-449b-bb3b-b64635956e7b' timed out waiting for connection from the Remote Spark Driver:33:1", 'io.netty.util.concurrent.AbstractFuture:get:AbstractFuture.java:41', 'org.apache.hive.spark.client.SparkClientImpl:<init>:SparkClientImpl.java:103', "*java.util.concurrent.TimeoutException:Client '0db2de9d-69b6-449b-bb3b-b64635956e7b' timed out waiting for connection from the Remote Spark Driver:8:6", 'org.apache.hive.spark.client.rpc.RpcServer$2:run:RpcServer.java:178', 'io.netty.util.concurrent.PromiseTask$RunnableAdapter:call:PromiseTask.java:38', 'io.netty.util.concurrent.ScheduledFutureTask:run:ScheduledFutureTask.java:120', 'io.netty.util.concurrent.AbstractEventExecutor:safeExecute:AbstractEventExecutor.java:163', 'io.netty.util.concurrent.SingleThreadEventExecutor:runAllTasks:SingleThreadEventExecutor.java:403', 'io.netty.channel.nio.NioEventLoop:run:NioEventLoop.java:463', 'io.netty.util.concurrent.SingleThreadEventExecutor$5:run:SingleThreadEventExecutor.java:858'], sqlState='42000', errorCode=30041, errorMessage="Error while processing statement: FAILED: Execution Error, return code 30041 from org.apache.hadoop.hive.ql.exec.spark.SparkTask. Failed to create Spark client for Spark session 0db2de9d-69b6-449b-bb3b-b64635956e7b: java.util.concurrent.TimeoutException: Client '0db2de9d-69b6-449b-bb3b-b64635956e7b' timed out waiting for connection from the Remote Spark Driver"), operationHandle=None)
2023-07-05 01:38:32,081 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-06 01:24:49,950 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-07 01:17:42,921 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-08 00:59:15,647 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-09 01:44:28,538 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-10 01:37:23,687 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-11 01:21:57,908 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-12 00:05:41,773 - ERROR - MainThread -fsk_hive_connect.py - execute_sql - 76 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while processing statement: FAILED: Execution Error, return code 30041 from org.apache.hadoop.hive.ql.exec.spark.SparkTask. Failed to create Spark client for Spark session 219bda3d-ebc8-4220-aade-2c16e10b1fd0: java.util.concurrent.TimeoutException: Client '219bda3d-ebc8-4220-aade-2c16e10b1fd0' timed out waiting for connection from the Remote Spark Driver:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:runQuery:SQLOperation.java:258', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:293', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.metadata.HiveException:Failed to create Spark client for Spark session 219bda3d-ebc8-4220-aade-2c16e10b1fd0: java.util.concurrent.TimeoutException: Client '219bda3d-ebc8-4220-aade-2c16e10b1fd0' timed out waiting for connection from the Remote Spark Driver:28:12", 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:getHiveException:SparkSessionImpl.java:296', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:open:SparkSessionImpl.java:140', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionManagerImpl:getSession:SparkSessionManagerImpl.java:131', 'org.apache.hadoop.hive.ql.exec.spark.SparkUtilities:getSparkSession:SparkUtilities.java:128', 'org.apache.hadoop.hive.ql.exec.spark.SparkTask:execute:SparkTask.java:122', 'org.apache.hadoop.hive.ql.exec.Task:executeTask:Task.java:199', 'org.apache.hadoop.hive.ql.exec.TaskRunner:runSequential:TaskRunner.java:97', 'org.apache.hadoop.hive.ql.Driver:launchTask:Driver.java:2250', 'org.apache.hadoop.hive.ql.Driver:execute:Driver.java:1893', 'org.apache.hadoop.hive.ql.Driver:runInternal:Driver.java:1613', 'org.apache.hadoop.hive.ql.Driver:run:Driver.java:1332', 'org.apache.hadoop.hive.ql.Driver:run:Driver.java:1327', 'org.apache.hive.service.cli.operation.SQLOperation:runQuery:SQLOperation.java:256', "*java.lang.RuntimeException:Timed out waiting for Remote Spark Driver to connect to HiveServer2.\nPossible reasons include network issues, errors in remote driver, cluster has no available resources, etc.\nPlease check YARN or Spark driver's logs for further information.:32:5", 'org.apache.hive.spark.client.SparkClientImpl:<init>:SparkClientImpl.java:124', 'org.apache.hive.spark.client.SparkClientFactory:createClient:SparkClientFactory.java:90', 'org.apache.hadoop.hive.ql.exec.spark.RemoteHiveSparkClient:createRemoteClient:RemoteHiveSparkClient.java:104', 'org.apache.hadoop.hive.ql.exec.spark.RemoteHiveSparkClient:<init>:RemoteHiveSparkClient.java:100', 'org.apache.hadoop.hive.ql.exec.spark.HiveSparkClientFactory:createHiveSparkClient:HiveSparkClientFactory.java:76', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:open:SparkSessionImpl.java:135', "*java.util.concurrent.ExecutionException:java.util.concurrent.TimeoutException: Client '219bda3d-ebc8-4220-aade-2c16e10b1fd0' timed out waiting for connection from the Remote Spark Driver:33:1", 'io.netty.util.concurrent.AbstractFuture:get:AbstractFuture.java:41', 'org.apache.hive.spark.client.SparkClientImpl:<init>:SparkClientImpl.java:103', "*java.util.concurrent.TimeoutException:Client '219bda3d-ebc8-4220-aade-2c16e10b1fd0' timed out waiting for connection from the Remote Spark Driver:8:6", 'org.apache.hive.spark.client.rpc.RpcServer$2:run:RpcServer.java:178', 'io.netty.util.concurrent.PromiseTask$RunnableAdapter:call:PromiseTask.java:38', 'io.netty.util.concurrent.ScheduledFutureTask:run:ScheduledFutureTask.java:120', 'io.netty.util.concurrent.AbstractEventExecutor:safeExecute:AbstractEventExecutor.java:163', 'io.netty.util.concurrent.SingleThreadEventExecutor:runAllTasks:SingleThreadEventExecutor.java:403', 'io.netty.channel.nio.NioEventLoop:run:NioEventLoop.java:463', 'io.netty.util.concurrent.SingleThreadEventExecutor$5:run:SingleThreadEventExecutor.java:858'], sqlState='42000', errorCode=30041, errorMessage="Error while processing statement: FAILED: Execution Error, return code 30041 from org.apache.hadoop.hive.ql.exec.spark.SparkTask. Failed to create Spark client for Spark session 219bda3d-ebc8-4220-aade-2c16e10b1fd0: java.util.concurrent.TimeoutException: Client '219bda3d-ebc8-4220-aade-2c16e10b1fd0' timed out waiting for connection from the Remote Spark Driver"), operationHandle=None)
2023-07-13 00:18:38,320 - ERROR - MainThread -fsk_hive_connect.py - execute_sql - 76 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while processing statement: FAILED: Execution Error, return code 30041 from org.apache.hadoop.hive.ql.exec.spark.SparkTask. Failed to create Spark client for Spark session 0b230a11-207e-4a0e-af43-ccbbc5a33f52: java.util.concurrent.TimeoutException: Client '0b230a11-207e-4a0e-af43-ccbbc5a33f52' timed out waiting for connection from the Remote Spark Driver:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:runQuery:SQLOperation.java:258', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:293', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.metadata.HiveException:Failed to create Spark client for Spark session 0b230a11-207e-4a0e-af43-ccbbc5a33f52: java.util.concurrent.TimeoutException: Client '0b230a11-207e-4a0e-af43-ccbbc5a33f52' timed out waiting for connection from the Remote Spark Driver:28:12", 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:getHiveException:SparkSessionImpl.java:296', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:open:SparkSessionImpl.java:140', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionManagerImpl:getSession:SparkSessionManagerImpl.java:131', 'org.apache.hadoop.hive.ql.exec.spark.SparkUtilities:getSparkSession:SparkUtilities.java:128', 'org.apache.hadoop.hive.ql.exec.spark.SparkTask:execute:SparkTask.java:122', 'org.apache.hadoop.hive.ql.exec.Task:executeTask:Task.java:199', 'org.apache.hadoop.hive.ql.exec.TaskRunner:runSequential:TaskRunner.java:97', 'org.apache.hadoop.hive.ql.Driver:launchTask:Driver.java:2250', 'org.apache.hadoop.hive.ql.Driver:execute:Driver.java:1893', 'org.apache.hadoop.hive.ql.Driver:runInternal:Driver.java:1613', 'org.apache.hadoop.hive.ql.Driver:run:Driver.java:1332', 'org.apache.hadoop.hive.ql.Driver:run:Driver.java:1327', 'org.apache.hive.service.cli.operation.SQLOperation:runQuery:SQLOperation.java:256', "*java.lang.RuntimeException:Timed out waiting for Remote Spark Driver to connect to HiveServer2.\nPossible reasons include network issues, errors in remote driver, cluster has no available resources, etc.\nPlease check YARN or Spark driver's logs for further information.:32:5", 'org.apache.hive.spark.client.SparkClientImpl:<init>:SparkClientImpl.java:124', 'org.apache.hive.spark.client.SparkClientFactory:createClient:SparkClientFactory.java:90', 'org.apache.hadoop.hive.ql.exec.spark.RemoteHiveSparkClient:createRemoteClient:RemoteHiveSparkClient.java:104', 'org.apache.hadoop.hive.ql.exec.spark.RemoteHiveSparkClient:<init>:RemoteHiveSparkClient.java:100', 'org.apache.hadoop.hive.ql.exec.spark.HiveSparkClientFactory:createHiveSparkClient:HiveSparkClientFactory.java:76', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:open:SparkSessionImpl.java:135', "*java.util.concurrent.ExecutionException:java.util.concurrent.TimeoutException: Client '0b230a11-207e-4a0e-af43-ccbbc5a33f52' timed out waiting for connection from the Remote Spark Driver:33:1", 'io.netty.util.concurrent.AbstractFuture:get:AbstractFuture.java:41', 'org.apache.hive.spark.client.SparkClientImpl:<init>:SparkClientImpl.java:103', "*java.util.concurrent.TimeoutException:Client '0b230a11-207e-4a0e-af43-ccbbc5a33f52' timed out waiting for connection from the Remote Spark Driver:8:6", 'org.apache.hive.spark.client.rpc.RpcServer$2:run:RpcServer.java:178', 'io.netty.util.concurrent.PromiseTask$RunnableAdapter:call:PromiseTask.java:38', 'io.netty.util.concurrent.ScheduledFutureTask:run:ScheduledFutureTask.java:120', 'io.netty.util.concurrent.AbstractEventExecutor:safeExecute:AbstractEventExecutor.java:163', 'io.netty.util.concurrent.SingleThreadEventExecutor:runAllTasks:SingleThreadEventExecutor.java:403', 'io.netty.channel.nio.NioEventLoop:run:NioEventLoop.java:463', 'io.netty.util.concurrent.SingleThreadEventExecutor$5:run:SingleThreadEventExecutor.java:858'], sqlState='42000', errorCode=30041, errorMessage="Error while processing statement: FAILED: Execution Error, return code 30041 from org.apache.hadoop.hive.ql.exec.spark.SparkTask. Failed to create Spark client for Spark session 0b230a11-207e-4a0e-af43-ccbbc5a33f52: java.util.concurrent.TimeoutException: Client '0b230a11-207e-4a0e-af43-ccbbc5a33f52' timed out waiting for connection from the Remote Spark Driver"), operationHandle=None)
2023-07-13 00:23:40,498 - ERROR - MainThread -fsk_hive_connect.py - execute_sql - 76 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while processing statement: FAILED: Execution Error, return code 30041 from org.apache.hadoop.hive.ql.exec.spark.SparkTask. Failed to create Spark client for Spark session f5e1b653-cbda-49ff-9d17-fa336ea8a57b: java.util.concurrent.TimeoutException: Client 'f5e1b653-cbda-49ff-9d17-fa336ea8a57b' timed out waiting for connection from the Remote Spark Driver:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:runQuery:SQLOperation.java:258', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:293', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.metadata.HiveException:Failed to create Spark client for Spark session f5e1b653-cbda-49ff-9d17-fa336ea8a57b: java.util.concurrent.TimeoutException: Client 'f5e1b653-cbda-49ff-9d17-fa336ea8a57b' timed out waiting for connection from the Remote Spark Driver:28:12", 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:getHiveException:SparkSessionImpl.java:296', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:open:SparkSessionImpl.java:140', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionManagerImpl:getSession:SparkSessionManagerImpl.java:131', 'org.apache.hadoop.hive.ql.exec.spark.SparkUtilities:getSparkSession:SparkUtilities.java:128', 'org.apache.hadoop.hive.ql.exec.spark.SparkTask:execute:SparkTask.java:122', 'org.apache.hadoop.hive.ql.exec.Task:executeTask:Task.java:199', 'org.apache.hadoop.hive.ql.exec.TaskRunner:runSequential:TaskRunner.java:97', 'org.apache.hadoop.hive.ql.Driver:launchTask:Driver.java:2250', 'org.apache.hadoop.hive.ql.Driver:execute:Driver.java:1893', 'org.apache.hadoop.hive.ql.Driver:runInternal:Driver.java:1613', 'org.apache.hadoop.hive.ql.Driver:run:Driver.java:1332', 'org.apache.hadoop.hive.ql.Driver:run:Driver.java:1327', 'org.apache.hive.service.cli.operation.SQLOperation:runQuery:SQLOperation.java:256', "*java.lang.RuntimeException:Timed out waiting for Remote Spark Driver to connect to HiveServer2.\nPossible reasons include network issues, errors in remote driver, cluster has no available resources, etc.\nPlease check YARN or Spark driver's logs for further information.:32:5", 'org.apache.hive.spark.client.SparkClientImpl:<init>:SparkClientImpl.java:124', 'org.apache.hive.spark.client.SparkClientFactory:createClient:SparkClientFactory.java:90', 'org.apache.hadoop.hive.ql.exec.spark.RemoteHiveSparkClient:createRemoteClient:RemoteHiveSparkClient.java:104', 'org.apache.hadoop.hive.ql.exec.spark.RemoteHiveSparkClient:<init>:RemoteHiveSparkClient.java:100', 'org.apache.hadoop.hive.ql.exec.spark.HiveSparkClientFactory:createHiveSparkClient:HiveSparkClientFactory.java:76', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:open:SparkSessionImpl.java:135', "*java.util.concurrent.ExecutionException:java.util.concurrent.TimeoutException: Client 'f5e1b653-cbda-49ff-9d17-fa336ea8a57b' timed out waiting for connection from the Remote Spark Driver:33:1", 'io.netty.util.concurrent.AbstractFuture:get:AbstractFuture.java:41', 'org.apache.hive.spark.client.SparkClientImpl:<init>:SparkClientImpl.java:103', "*java.util.concurrent.TimeoutException:Client 'f5e1b653-cbda-49ff-9d17-fa336ea8a57b' timed out waiting for connection from the Remote Spark Driver:8:6", 'org.apache.hive.spark.client.rpc.RpcServer$2:run:RpcServer.java:178', 'io.netty.util.concurrent.PromiseTask$RunnableAdapter:call:PromiseTask.java:38', 'io.netty.util.concurrent.ScheduledFutureTask:run:ScheduledFutureTask.java:120', 'io.netty.util.concurrent.AbstractEventExecutor:safeExecute:AbstractEventExecutor.java:163', 'io.netty.util.concurrent.SingleThreadEventExecutor:runAllTasks:SingleThreadEventExecutor.java:403', 'io.netty.channel.nio.NioEventLoop:run:NioEventLoop.java:463', 'io.netty.util.concurrent.SingleThreadEventExecutor$5:run:SingleThreadEventExecutor.java:858'], sqlState='42000', errorCode=30041, errorMessage="Error while processing statement: FAILED: Execution Error, return code 30041 from org.apache.hadoop.hive.ql.exec.spark.SparkTask. Failed to create Spark client for Spark session f5e1b653-cbda-49ff-9d17-fa336ea8a57b: java.util.concurrent.TimeoutException: Client 'f5e1b653-cbda-49ff-9d17-fa336ea8a57b' timed out waiting for connection from the Remote Spark Driver"), operationHandle=None)
2023-07-14 01:23:16,912 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-15 01:19:44,662 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-16 00:28:54,158 - ERROR - MainThread -fsk_hive_connect.py - execute_sql - 76 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while processing statement: FAILED: Execution Error, return code 30041 from org.apache.hadoop.hive.ql.exec.spark.SparkTask. Failed to create Spark client for Spark session 756e69f2-bfbf-4ca5-b2c3-8d81387ba65a: java.util.concurrent.TimeoutException: Client '756e69f2-bfbf-4ca5-b2c3-8d81387ba65a' timed out waiting for connection from the Remote Spark Driver:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:runQuery:SQLOperation.java:258', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:293', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.metadata.HiveException:Failed to create Spark client for Spark session 756e69f2-bfbf-4ca5-b2c3-8d81387ba65a: java.util.concurrent.TimeoutException: Client '756e69f2-bfbf-4ca5-b2c3-8d81387ba65a' timed out waiting for connection from the Remote Spark Driver:28:12", 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:getHiveException:SparkSessionImpl.java:296', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:open:SparkSessionImpl.java:140', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionManagerImpl:getSession:SparkSessionManagerImpl.java:131', 'org.apache.hadoop.hive.ql.exec.spark.SparkUtilities:getSparkSession:SparkUtilities.java:128', 'org.apache.hadoop.hive.ql.exec.spark.SparkTask:execute:SparkTask.java:122', 'org.apache.hadoop.hive.ql.exec.Task:executeTask:Task.java:199', 'org.apache.hadoop.hive.ql.exec.TaskRunner:runSequential:TaskRunner.java:97', 'org.apache.hadoop.hive.ql.Driver:launchTask:Driver.java:2250', 'org.apache.hadoop.hive.ql.Driver:execute:Driver.java:1893', 'org.apache.hadoop.hive.ql.Driver:runInternal:Driver.java:1613', 'org.apache.hadoop.hive.ql.Driver:run:Driver.java:1332', 'org.apache.hadoop.hive.ql.Driver:run:Driver.java:1327', 'org.apache.hive.service.cli.operation.SQLOperation:runQuery:SQLOperation.java:256', "*java.lang.RuntimeException:Timed out waiting for Remote Spark Driver to connect to HiveServer2.\nPossible reasons include network issues, errors in remote driver, cluster has no available resources, etc.\nPlease check YARN or Spark driver's logs for further information.:32:5", 'org.apache.hive.spark.client.SparkClientImpl:<init>:SparkClientImpl.java:124', 'org.apache.hive.spark.client.SparkClientFactory:createClient:SparkClientFactory.java:90', 'org.apache.hadoop.hive.ql.exec.spark.RemoteHiveSparkClient:createRemoteClient:RemoteHiveSparkClient.java:104', 'org.apache.hadoop.hive.ql.exec.spark.RemoteHiveSparkClient:<init>:RemoteHiveSparkClient.java:100', 'org.apache.hadoop.hive.ql.exec.spark.HiveSparkClientFactory:createHiveSparkClient:HiveSparkClientFactory.java:76', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:open:SparkSessionImpl.java:135', "*java.util.concurrent.ExecutionException:java.util.concurrent.TimeoutException: Client '756e69f2-bfbf-4ca5-b2c3-8d81387ba65a' timed out waiting for connection from the Remote Spark Driver:33:1", 'io.netty.util.concurrent.AbstractFuture:get:AbstractFuture.java:41', 'org.apache.hive.spark.client.SparkClientImpl:<init>:SparkClientImpl.java:103', "*java.util.concurrent.TimeoutException:Client '756e69f2-bfbf-4ca5-b2c3-8d81387ba65a' timed out waiting for connection from the Remote Spark Driver:8:6", 'org.apache.hive.spark.client.rpc.RpcServer$2:run:RpcServer.java:178', 'io.netty.util.concurrent.PromiseTask$RunnableAdapter:call:PromiseTask.java:38', 'io.netty.util.concurrent.ScheduledFutureTask:run:ScheduledFutureTask.java:120', 'io.netty.util.concurrent.AbstractEventExecutor:safeExecute:AbstractEventExecutor.java:163', 'io.netty.util.concurrent.SingleThreadEventExecutor:runAllTasks:SingleThreadEventExecutor.java:403', 'io.netty.channel.nio.NioEventLoop:run:NioEventLoop.java:463', 'io.netty.util.concurrent.SingleThreadEventExecutor$5:run:SingleThreadEventExecutor.java:858'], sqlState='42000', errorCode=30041, errorMessage="Error while processing statement: FAILED: Execution Error, return code 30041 from org.apache.hadoop.hive.ql.exec.spark.SparkTask. Failed to create Spark client for Spark session 756e69f2-bfbf-4ca5-b2c3-8d81387ba65a: java.util.concurrent.TimeoutException: Client '756e69f2-bfbf-4ca5-b2c3-8d81387ba65a' timed out waiting for connection from the Remote Spark Driver"), operationHandle=None)
2023-07-16 00:33:56,985 - ERROR - MainThread -fsk_hive_connect.py - execute_sql - 76 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while processing statement: FAILED: Execution Error, return code 30041 from org.apache.hadoop.hive.ql.exec.spark.SparkTask. Failed to create Spark client for Spark session e4cac8e9-931b-4c9d-a89b-9792531513d3: java.util.concurrent.TimeoutException: Client 'e4cac8e9-931b-4c9d-a89b-9792531513d3' timed out waiting for connection from the Remote Spark Driver:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:runQuery:SQLOperation.java:258', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:293', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.metadata.HiveException:Failed to create Spark client for Spark session e4cac8e9-931b-4c9d-a89b-9792531513d3: java.util.concurrent.TimeoutException: Client 'e4cac8e9-931b-4c9d-a89b-9792531513d3' timed out waiting for connection from the Remote Spark Driver:28:12", 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:getHiveException:SparkSessionImpl.java:296', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:open:SparkSessionImpl.java:140', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionManagerImpl:getSession:SparkSessionManagerImpl.java:131', 'org.apache.hadoop.hive.ql.exec.spark.SparkUtilities:getSparkSession:SparkUtilities.java:128', 'org.apache.hadoop.hive.ql.exec.spark.SparkTask:execute:SparkTask.java:122', 'org.apache.hadoop.hive.ql.exec.Task:executeTask:Task.java:199', 'org.apache.hadoop.hive.ql.exec.TaskRunner:runSequential:TaskRunner.java:97', 'org.apache.hadoop.hive.ql.Driver:launchTask:Driver.java:2250', 'org.apache.hadoop.hive.ql.Driver:execute:Driver.java:1893', 'org.apache.hadoop.hive.ql.Driver:runInternal:Driver.java:1613', 'org.apache.hadoop.hive.ql.Driver:run:Driver.java:1332', 'org.apache.hadoop.hive.ql.Driver:run:Driver.java:1327', 'org.apache.hive.service.cli.operation.SQLOperation:runQuery:SQLOperation.java:256', "*java.lang.RuntimeException:Timed out waiting for Remote Spark Driver to connect to HiveServer2.\nPossible reasons include network issues, errors in remote driver, cluster has no available resources, etc.\nPlease check YARN or Spark driver's logs for further information.:32:5", 'org.apache.hive.spark.client.SparkClientImpl:<init>:SparkClientImpl.java:124', 'org.apache.hive.spark.client.SparkClientFactory:createClient:SparkClientFactory.java:90', 'org.apache.hadoop.hive.ql.exec.spark.RemoteHiveSparkClient:createRemoteClient:RemoteHiveSparkClient.java:104', 'org.apache.hadoop.hive.ql.exec.spark.RemoteHiveSparkClient:<init>:RemoteHiveSparkClient.java:100', 'org.apache.hadoop.hive.ql.exec.spark.HiveSparkClientFactory:createHiveSparkClient:HiveSparkClientFactory.java:76', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:open:SparkSessionImpl.java:135', "*java.util.concurrent.ExecutionException:java.util.concurrent.TimeoutException: Client 'e4cac8e9-931b-4c9d-a89b-9792531513d3' timed out waiting for connection from the Remote Spark Driver:33:1", 'io.netty.util.concurrent.AbstractFuture:get:AbstractFuture.java:41', 'org.apache.hive.spark.client.SparkClientImpl:<init>:SparkClientImpl.java:103', "*java.util.concurrent.TimeoutException:Client 'e4cac8e9-931b-4c9d-a89b-9792531513d3' timed out waiting for connection from the Remote Spark Driver:8:6", 'org.apache.hive.spark.client.rpc.RpcServer$2:run:RpcServer.java:178', 'io.netty.util.concurrent.PromiseTask$RunnableAdapter:call:PromiseTask.java:38', 'io.netty.util.concurrent.ScheduledFutureTask:run:ScheduledFutureTask.java:120', 'io.netty.util.concurrent.AbstractEventExecutor:safeExecute:AbstractEventExecutor.java:163', 'io.netty.util.concurrent.SingleThreadEventExecutor:runAllTasks:SingleThreadEventExecutor.java:403', 'io.netty.channel.nio.NioEventLoop:run:NioEventLoop.java:463', 'io.netty.util.concurrent.SingleThreadEventExecutor$5:run:SingleThreadEventExecutor.java:858'], sqlState='42000', errorCode=30041, errorMessage="Error while processing statement: FAILED: Execution Error, return code 30041 from org.apache.hadoop.hive.ql.exec.spark.SparkTask. Failed to create Spark client for Spark session e4cac8e9-931b-4c9d-a89b-9792531513d3: java.util.concurrent.TimeoutException: Client 'e4cac8e9-931b-4c9d-a89b-9792531513d3' timed out waiting for connection from the Remote Spark Driver"), operationHandle=None)
2023-07-17 01:16:59,052 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-18 01:17:06,641 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-19 01:06:12,300 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-20 00:53:34,103 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-21 01:39:14,348 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-22 01:35:06,090 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-23 00:35:58,969 - ERROR - MainThread -fsk_hive_connect.py - execute_sql - 76 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while processing statement: FAILED: Execution Error, return code 30041 from org.apache.hadoop.hive.ql.exec.spark.SparkTask. Failed to create Spark client for Spark session 8712b5ef-56b4-4d74-b5df-1349bf07460d: java.util.concurrent.TimeoutException: Client '8712b5ef-56b4-4d74-b5df-1349bf07460d' timed out waiting for connection from the Remote Spark Driver:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:runQuery:SQLOperation.java:258', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:293', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.metadata.HiveException:Failed to create Spark client for Spark session 8712b5ef-56b4-4d74-b5df-1349bf07460d: java.util.concurrent.TimeoutException: Client '8712b5ef-56b4-4d74-b5df-1349bf07460d' timed out waiting for connection from the Remote Spark Driver:28:12", 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:getHiveException:SparkSessionImpl.java:296', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:open:SparkSessionImpl.java:140', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionManagerImpl:getSession:SparkSessionManagerImpl.java:131', 'org.apache.hadoop.hive.ql.exec.spark.SparkUtilities:getSparkSession:SparkUtilities.java:128', 'org.apache.hadoop.hive.ql.exec.spark.SparkTask:execute:SparkTask.java:122', 'org.apache.hadoop.hive.ql.exec.Task:executeTask:Task.java:199', 'org.apache.hadoop.hive.ql.exec.TaskRunner:runSequential:TaskRunner.java:97', 'org.apache.hadoop.hive.ql.Driver:launchTask:Driver.java:2250', 'org.apache.hadoop.hive.ql.Driver:execute:Driver.java:1893', 'org.apache.hadoop.hive.ql.Driver:runInternal:Driver.java:1613', 'org.apache.hadoop.hive.ql.Driver:run:Driver.java:1332', 'org.apache.hadoop.hive.ql.Driver:run:Driver.java:1327', 'org.apache.hive.service.cli.operation.SQLOperation:runQuery:SQLOperation.java:256', "*java.lang.RuntimeException:Timed out waiting for Remote Spark Driver to connect to HiveServer2.\nPossible reasons include network issues, errors in remote driver, cluster has no available resources, etc.\nPlease check YARN or Spark driver's logs for further information.:32:5", 'org.apache.hive.spark.client.SparkClientImpl:<init>:SparkClientImpl.java:124', 'org.apache.hive.spark.client.SparkClientFactory:createClient:SparkClientFactory.java:90', 'org.apache.hadoop.hive.ql.exec.spark.RemoteHiveSparkClient:createRemoteClient:RemoteHiveSparkClient.java:104', 'org.apache.hadoop.hive.ql.exec.spark.RemoteHiveSparkClient:<init>:RemoteHiveSparkClient.java:100', 'org.apache.hadoop.hive.ql.exec.spark.HiveSparkClientFactory:createHiveSparkClient:HiveSparkClientFactory.java:76', 'org.apache.hadoop.hive.ql.exec.spark.session.SparkSessionImpl:open:SparkSessionImpl.java:135', "*java.util.concurrent.ExecutionException:java.util.concurrent.TimeoutException: Client '8712b5ef-56b4-4d74-b5df-1349bf07460d' timed out waiting for connection from the Remote Spark Driver:33:1", 'io.netty.util.concurrent.AbstractFuture:get:AbstractFuture.java:41', 'org.apache.hive.spark.client.SparkClientImpl:<init>:SparkClientImpl.java:103', "*java.util.concurrent.TimeoutException:Client '8712b5ef-56b4-4d74-b5df-1349bf07460d' timed out waiting for connection from the Remote Spark Driver:8:6", 'org.apache.hive.spark.client.rpc.RpcServer$2:run:RpcServer.java:178', 'io.netty.util.concurrent.PromiseTask$RunnableAdapter:call:PromiseTask.java:38', 'io.netty.util.concurrent.ScheduledFutureTask:run:ScheduledFutureTask.java:120', 'io.netty.util.concurrent.AbstractEventExecutor:safeExecute:AbstractEventExecutor.java:163', 'io.netty.util.concurrent.SingleThreadEventExecutor:runAllTasks:SingleThreadEventExecutor.java:403', 'io.netty.channel.nio.NioEventLoop:run:NioEventLoop.java:463', 'io.netty.util.concurrent.SingleThreadEventExecutor$5:run:SingleThreadEventExecutor.java:858'], sqlState='42000', errorCode=30041, errorMessage="Error while processing statement: FAILED: Execution Error, return code 30041 from org.apache.hadoop.hive.ql.exec.spark.SparkTask. Failed to create Spark client for Spark session 8712b5ef-56b4-4d74-b5df-1349bf07460d: java.util.concurrent.TimeoutException: Client '8712b5ef-56b4-4d74-b5df-1349bf07460d' timed out waiting for connection from the Remote Spark Driver"), operationHandle=None)
2023-07-24 01:40:25,323 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-25 01:35:06,246 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-26 01:43:54,017 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-27 01:27:15,975 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-28 01:23:48,216 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-29 01:32:52,368 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-30 01:58:29,469 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
2023-07-31 01:51:01,500 - ERROR - MainThread -fsk_hive_connect.py - excute_sql2df - 131 - TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=["*org.apache.hive.service.cli.HiveSQLException:Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:17:16", 'org.apache.hive.service.cli.operation.Operation:toSQLException:Operation.java:335', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:207', 'org.apache.hive.service.cli.operation.SQLOperation:runInternal:SQLOperation.java:290', 'org.apache.hive.service.cli.operation.Operation:run:Operation.java:266', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatementInternal:HiveSessionImpl.java:504', 'org.apache.hive.service.cli.session.HiveSessionImpl:executeStatement:HiveSessionImpl.java:479', 'org.apache.hive.service.cli.CLIService:executeStatement:CLIService.java:270', 'org.apache.hive.service.cli.thrift.ThriftCLIService:ExecuteStatement:ThriftCLIService.java:508', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1437', 'org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement:getResult:TCLIService.java:1422', 'org.apache.thrift.ProcessFunction:process:ProcessFunction.java:39', 'org.apache.thrift.TBaseProcessor:process:TBaseProcessor.java:39', 'org.apache.hive.service.auth.TSetIpAddressProcessor:process:TSetIpAddressProcessor.java:56', 'org.apache.thrift.server.TThreadPoolServer$WorkerProcess:run:TThreadPoolServer.java:286', 'java.util.concurrent.ThreadPoolExecutor:runWorker:ThreadPoolExecutor.java:1149', 'java.util.concurrent.ThreadPoolExecutor$Worker:run:ThreadPoolExecutor.java:624', 'java.lang.Thread:run:Thread.java:748', "*org.apache.hadoop.hive.ql.parse.ParseException:line 13:28 cannot recognize input near 'in' '(' ')' in expression specification:22:6", 'org.apache.hadoop.hive.ql.parse.ParseDriver:parse:ParseDriver.java:221', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:75', 'org.apache.hadoop.hive.ql.parse.ParseUtils:parse:ParseUtils.java:68', 'org.apache.hadoop.hive.ql.Driver:compile:Driver.java:561', 'org.apache.hadoop.hive.ql.Driver:compileInternal:Driver.java:1414', 'org.apache.hadoop.hive.ql.Driver:compileAndRespond:Driver.java:1391', 'org.apache.hive.service.cli.operation.SQLOperation:prepare:SQLOperation.java:205'], sqlState='42000', errorCode=40000, errorMessage="Error while compiling statement: FAILED: ParseException line 13:28 cannot recognize input near 'in' '(' ')' in expression specification"), operationHandle=None)
